DATA/production/qc-async/tpc.json

```diff
@@ -93,5 +93,20 @@
     }
   },
   "dataSamplingPolicies": [
+    {
+      "id": "tpc-clusters",
+      "active": "true",
+      "machines": ["localhost"],
+      "query" : "inputClusters:TPC/CLUSTERNATIVE",
+      "outputs": "sampled-clusters:DS/CLUSTERNATIVE",
```

Comment:
Hi @tklemenz , 
Is this `outputs` needed? I have never seen it, maybe it is optional? @knopers8 .

Reply:
Hi Chiara, 
I tested it locally and it did not work without it. That was actually the option I was missing. The `clusterHandler` taking care of the cluster input is looking for the origin `DS` to not mix it up with non-sampled data. Afaik the `outputs` is just re-labeling the messages in the DPL. But I guess Piotr has more insight.

Reply:
Ok, probably it is because you look for this specific origin. I will merge now, if @knopers8 can comment further, I could learn :-)